docs/docs_source/postmessage.rst

```diff
@@ -245,6 +245,17 @@ The host page receives the following messages; all others are ignored:
 * :data:`UI_FileVersions`
 * :data:`UI_Sharing`
 
+As an addition to message-specific values Office Online sends a set of common values on every PostMessage request it makes:
+
+	.. attribute:: Common Values
```

Comment:
Let's just add a new sub-heading for these common values. So something like:

    Common Values
    ~~~~~~~~~~~~~

    In addition to message-specific values passed with each message, Office Online
    sends the following common values on every outgoing PostMessage:

    ...